transport/src/androidTest/kotlin/com/genesys/cloud/messenger/transport/util/Request.kt

```diff
@@ -0,0 +1,8 @@
+package com.genesys.cloud.messenger.transport.util
+
+internal object Request {
+    const val configureRequest =
+        """{"token":"00000000-0000-0000-0000-000000000000","deploymentId":"deploymentId","journeyContext":{"customer":{"id":"00000000-0000-0000-0000-000000000000","idType":"cookie"},"customerSession":{"id":"","type":"web"}},"action":"configureSession"}"""
+    const val userTypingRequest =
+        """{"token":"00000000-0000-0000-0000-000000000000","action":"onMessage","message":{"events":[{"eventType":"Typing","typing":{"type":"On","duration":5000}}],"type":"Event"}}"""
```

Comment:
Was it agreed by Shyrka team that the "duration" is included in the typing indicator events for both customer and agent typing? (only asking because it's not in the Developer Center documentation)

Reply:
I have raised this question in the `typing indicator` chatroom, but was ignored. I will dm Julien from Shyrka. usually he is the person who deals with their documentation and is very helpful.

Reply:
[> Was it agreed by Shyrka team that the "duration" is included in the typing indicator events for both customer and agent typing? (only asking because it's not in the Developer Center documentation)]
So, after discussion with Shyrka team, it was decided that:
1. `duration` field should be added to the [agent typing response](https://bitbucket.org/inindca/developer-center/pull-requests/1131/shyrka-1627-add-duration-to-typing-event?link_source=email). There are a [pr opened](https://bitbucket.org/inindca/developer-center/pull-requests/1131/shyrka-1627-add-duration-to-typing-event?link_source=email)
2. The `"type":"On"` must be included in [customer typing request](https://developer.genesys.cloud/commdigital/digital/webmessaging/websocketapi#customer-typing).
3. `duration` should be removed from [customer typing request](https://developer.genesys.cloud/commdigital/digital/webmessaging/websocketapi#customer-typing).
4. `duration` must be an optional value in agent typing response with fallback value of 5000ms if not set. 

Going to update this pr accordingly with following changes:
1.  Remove `duration` from user typing request.
2. Make `duration` a nullable with fallback value of 5000 ms if not present.